src/openforms/submissions/tests/factories.py

```diff
@@ -122,6 +122,7 @@ def from_components(
         cls,
         components_list: List[dict],
         submitted_data: dict = None,
+        form_definition_slug: str = "",
```

Comment:
how about `form_definition_kwargs = None`, and then in the body:

```py
form_definition_kwargs = form_definition_kwargs or {}
form_definition_kwargs.setdefault("name", ...)
form_definition_kwargs.setdefault("configuration", ...)
```

and now you can pass generic factory kwargs while still being functional without any kwargs